src/io/scratchblocks/toScratchblocks.ts

```diff
@@ -1,28 +1,29 @@
 import Project from "../../Project";
 import Script from "../../Script";
 import Target from "../../Target";
-import Block from "../../Block";
+import Block, { KnownBlock } from "../../Block";
 import * as BlockInput from "../../BlockInput";
 import { OpCode } from "../../OpCode";
 
-interface ToScratchblocksOptions {
+export interface ToScratchblocksOptions {
   indent: string;
 }
 
 export default function toScratchblocks(
+  project: Project,
   options: Partial<ToScratchblocksOptions> = {}
-): { [targetName: string]: string } {
-  const project: Project = this;
-
+): {
+  [targetName: string]: string;
+} {
   const defaultOptions: ToScratchblocksOptions = {
     indent: "\t"
   };
-  options = { ...defaultOptions, ...options };
+  const fullOptions = { ...defaultOptions, ...options };
```

Comment:
Can we change this to use the usual naming scheme, with `inOptions: Partial<...>` and `const options = { ...defaultOptions, ...inOptions }`?

Reply:
I'll do so